OrderApi/Models/OrderItem.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ShoesOnContainers.Services.OrderApi.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShoesOnContainers.Services.OrderApi.Models
+{
+    public class OrderItem
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+         
+         public string ProductName { get; set; }
```

Comment:
do we want ProductName or EventName? Our other services refer to it as EventName and EventId. But you would have to update it everywhere in OrderApi from ProductName to EventName if you want to change it.